Dockerfile

```diff
@@ -9,7 +9,8 @@ RUN apt-get update && apt-get install -y \
     && docker-php-ext-install -j$(nproc) iconv \
     && docker-php-ext-configure gd --with-freetype=/usr/include/ --with-jpeg=/usr/include/ \
     && docker-php-ext-configure pgsql -with-pgsql=/usr/local/pgsql \
-    && docker-php-ext-install -j$(nproc) gd pdo pdo_mysql pdo_pgsql pgsql
+    && docker-php-ext-install -j$(nproc) gd pdo pdo_mysql pdo_pgsql pgsql \
+    && rm -rf /var/lib/apt/lists/*
```

Comment:
```suggestion
    && rm -rf \
        /var/lib/apt/lists/* \
        /tmp/* \
        /var/tmp/*
```

Reply:
`apt-get clean` is actually not needed :smile:

Reply:
Why?

According to the https://manpages.ubuntu.com/manpages/lunar/en/man8/apt-get.8.html documentation page, the `apt-get clean` command removes package files from:
- `/var/cache/apt/archives/`
- `/var/cache/apt/archives/partial/`

directories that have been cleared neither by `rm -rf` nor another command 🙃

Reply:
i checked it in the image. /tmp and /var/tmp are empty. also the cache dir is slim.
it's not worth deleting more than `/var/lib/apt/lists/*`